src/zrc/api/scopes.py

```diff
@@ -51,3 +51,13 @@
 * zaakobjectdetails op te vragen
 """
 )
+
+
+SCOPE_ZAKEN_ALLES_VERWIJDEREN = Scope(
+    'zds.scopes.zaken.verwijderen',
+    description="""
+**Laat toe om**:
+
+* zaken te verwijderen
```

Comment:
I feel like we should list all the resulting cascading deletes as well, no?

Reply:
Well, I don't know about this. It doesn't allow you to delete individual other resources than Zaken. Deleting a Zaak just causes other objects to be deleted.

Reply:
the cascade delete definitely needs to be described somewhere, maybe better to put it in the viewset docstring under the `delete:` section. I would not be surprised of people using MySQL with MyISAM that doesn't have referential integrity, or no-SQL databases without referential integrity to rely on cascading deletes - this behaviour needs to be made explicit.

Reply:
Agree, hence it says:

```
    delete:
    Verwijdert een zaak, samen met alle gerelateerde resources binnen deze API.
```